docs/feature_combo.md

```diff
@@ -71,26 +92,118 @@ void process_combo_event(uint16_t combo_index, bool pressed) {
 }
 ```
 
-This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V.  But you could change this to do stuff like change layers, play sounds, or change settings.
-
-## Additional Configuration
-
-If you're using long combos, or even longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing.
-
-In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
-
-You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V. But you could change this to do stuff like play sounds or change settings.
 
-## Keycodes 
+# Advanced Configuration
+These configuration settings can be set in your `config.h` file.
 
-You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. 
+## Keycodes
```

Comment:
I feel like it can be kind of confusing to place the “Keycodes” subsection within the “Advanced Configuration” since it is said that “[all the below] configuration settings can be set in your `config.h` file.” Now, I know you've mentioned putting those keycodes in `keymap.c` but that contradicts with what's said in the beginning of this section. My suggestion would thus be to move the “Keycodes” subsection out of the “Advanced Configuration” section.